docs/resources/client/models/modelloaders.md

```diff
@@ -0,0 +1,444 @@
+# Custom Model Loaders
+
+A model is simply a shape. It can be a cube, a collection of cubes, a collection of triangles, or any other geometrical shape (or collection of geometrical shape). For most contexts, it is not relevant how a model is defined, as everything will end up as a `BakedModel` in memory anyway. As such, NeoForge adds the ability to register custom model loaders that can transform any model you want into a `BakedModel` for the game to use.
+
+The entry point for a block model remains the model JSON file. However, you can specify a `loader` field in the root of the JSON that will swap out the default loader for your own loader. A custom model loader may ignore all fields the default loader requires.
+
+## Builtin Model Loaders
+
+Besides the default model loader, NeoForge offers a total of seven builtin loaders, each serving a different purpose.
+
+### Composite Model
+
+A composite model can be used to specify different model parts in the parent and only apply some of them in a child. This is best illustrated by an example. Consider the following parent model at `examplemod:example_composite_model`:
```

Comment:
Composite models also can have different render types for each part, correct? Also, render order if I'm not mistaken.

Reply:
I did a quick glance through the code and found nothing like that? You can definitely specify render types per part, yes, as they're simply models.

Reply:
Tbh, I don't remember where I was going with this, so you can resolve it.